### PR TITLE
Add substitution: LayerNorm --> LayerNorm+BatchNorm

### PR DESCRIPTION
**Pull request by elad-c**
A substitution to move LayerNorm weights (gamma and beta) to an appended BatchNorm.
The substitution adds a BatchNorm after the LayerNorm with gamma & beta copied from LayerNorm.
Then, LayerNorm gamma & beta are set to 1 & 0 respectively.
BatchNorm epsilon, moving_avg & moving_var are set to 0, 0 & 1 respectively.

### CODE REVIEW

File: model_compression_toolkit/core/keras/graph_substitutions/substitutions/layer_norm_decomposition.py [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+    """
+
+    def __init__(self):
+        """
+        Matches LayerNorm node.
+        """
+        super().__init__(matcher_instance=NodeOperationMatcher(LayerNormalization))
+
+    @staticmethod
+    def _get_weight_by_name(mha_node, w_str):
```
Comment (haihabi) [inline model_compression_toolkit/core/keras/graph_substitutions/substitutions/layer_norm_decomposition.py:44]: 1. Comment 
2. Rename mha_node to something else

File: tests/keras_tests/feature_networks_tests/feature_networks/layer_norm_substitution_test.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
+# Copyright 2021 Sony Semiconductors Israel, Inc. All rights reserved.
```
Comment (haihabi) [inline tests/keras_tests/feature_networks_tests/feature_networks/layer_norm_substitution_test.py:1]: This is new file and the year is 2022
